src/ngMessages/messages.js

```diff
@@ -24,45 +24,64 @@ var jqLite = angular.element;
  * `ngMessage` and `ngMessageExp` directives.
  *
  * # Usage
- * The `ngMessages` directive listens on a key/value collection which is set on the ngMessages attribute.
- * Since the {@link ngModel ngModel} directive exposes an `$error` object, this error object can be
- * used with `ngMessages` to display control error messages in an easier way than with just regular angular
- * template directives.
+ * The `ngMessages` directive allows individual keys in a key/value collection to each be associated with an element
```

Comment:
"individual" and "each" can be dropped. 
I would also say "associated with a child element"